tests/integration/api/auth/backend/test_kerberos_auth.py

```diff
@@ -46,19 +46,17 @@ def app_for_kerberos():
         yield app.create_app(testing=True)
 
 
-@pytest.fixture(scope="module", autouse=True)
+@pytest.fixture(scope="module")
```

Comment:
I guess this is accidental fix that was needed? Can we maybe separate it out to another PR (just for trackability)?

Reply:
I move this usage into class fixture.

```python
    def _set_attrs(self, app_for_kerberos, dagbag_to_db):
```

Reply:
I see. Right. Should be maybe change the name of the class fixture? to `initialize` or something? 
I was under the impressuion (due to `_set_attrs` name) it was doing something different - and now it is not really setting the attributes.

Reply:
Yeah. This is a good point.

BTW we have quite a few tests classes where we use method (function) scoped fixture. And most of them has different name, so might be also a good point to make them more or less consistent.